src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemAlignmentUtils.java

```diff
@@ -15,34 +13,38 @@
 public class BwaMemAlignmentUtils {
     /**
      * Transforms an unaligned GATKRead into an aligned one, as specified by the supplied BwaMemAlignment.
+     * qualsArg can be null.
+     * readGroup can be null.
      *
      * Also, BWA does this odd thing, supplying AS and XS tags for unaligned reads.
      * To produce SAM records that are byte-identical to the ones created by the bwa mem command line,
      * call this method with alwaysGenerateASandXS=true.
      */
-    public static GATKRead applyAlignment(final GATKRead unalignedRead, final BwaMemAlignment alignment,
-                                            final List<String> refNames, final SAMFileHeader header,
-                                            final boolean softclipAlts, final boolean alwaysGenerateASandXS ) {
+    public static SAMRecord applyAlignment( final String readName, final byte[] basesArg, final byte[] qualsArg,
```

Comment:
Comment is out of date, and I'll fix it.  The reason it's a SAMRecord that I'm returning is that I'm writing a SAM file for the variant caller to use, so this is maximally convenient for me.  It's a one-liner to boost a SAMRecord into a GATKRead (by wrapping it with an adapter), so other users of the utility shouldn't find it too burdensome.  Currently there's only one other user, the BwaSparkEngine.